packages/web-components/package.json

```diff
@@ -92,7 +92,7 @@
     "ts-loader": "^7.0.2",
     "ts-node": "^8.0.0",
     "tsconfig-paths": "^3.9.0",
-    "typescript": "^3.9.0",
+    "typescript": "3.9.7",
```

Comment:
This seems fine - in the future anything which is intended to fix a React problem should likely not require a change here. There's no dependency on React for these packages and so I'd expect changes to be scoped to only those necessary.

Reply:
no worries, this has nothing to do with React. It's a 1st step to start using single version policy (and exact versions) for devDeps with a possibility to opt out. I could left web-components part of the repo as is but to prevent any kinds of version confusion/mismatches I set the version to be explicit.